doc/release/new-policy.rst

```diff
@@ -0,0 +1,159 @@
+Release Policy Project
+======================
+
+Summary
+-------
+
+Tarantool release policy changes to become more clear and intuitive.
+The new policy uses a `SemVer-like <https://semver.org/>`__ versioning format,
+and introduces a new version lifecycle with more long-time support series.
+This document explains the new release policy, versioning rules, and release series lifecycle.
+
+The new release policy replaces :doc:`the legacy policy <release/policy>`
+for the versions in 2.x.y series since 2.TBD.0,
+and for the future major series ones (3.0.0+) as well.
+
+The topics below describe the new versioning policy in more detail.
+
+Versioning
+----------
+
+Production-ready (release) version labels consist of major, minor and patch numbers:
+
+..  code-block:: text
+
+    MAJOR.MINOR.PATCH
+
+Versions in development and release candidates use the same pattern with an additional suffix:
+
+..  code-block:: text
+
+    MAJOR.MINOR.PATCH-<pre-release suffix>
+
+There are four stages of development before a release:
+
+#.  Alpha (``MAJOR.MINOR.PATCH-alphaN``)
+#.  Beta (``MAJOR.MINOR.PATCH-betaN``)
+#.  Release candidate (``MAJOR.MINOR.PATCH-rcN``)
+#.  Nightly build (``MAJOR.MINOR.PATCH-dev``)
```

Comment:
We call 'Early development', 'Support' and 'End of life' as stages, so, maybe, it would be better to call pre-release / build types using another word.